src/app/codegen-data-model/tests/TestCodegenModelViaMocks.cpp

```diff
@@ -2430,6 +2435,41 @@ TEST(TestCodegenModelViaMocks, EmberWriteAttributeAccessInterfaceTest)
     TestEmberScalarNullWrite<int64_t, ZCL_INT64S_ATTRIBUTE_TYPE>();
 }
 
+TEST(TestCodegenModelViaMocks, EmberInvokeTest)
+{
+    // Ember invoke is extrealy hard-coded, so generally we just need to validate that
```

Comment:
nit, what is extrealy hard-coded? Do you mean externally hard-coded?

Reply:
I think I meant `extreamly`. Re-phrased.